test-gui-e2e-setup.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -exuo pipefail
+
+###########################
+# Set up end to end tests #
+###########################
+# 1. Define and export ARTIFACT_DIR
```

Comment:
The script below is pretty clear, not sure if these comments add any real value or clarification.